### PR TITLE
Benchmark metrics

### PR DESCRIPTION
**Pull request by javaThonc**
update the effect of Tabnet on Alpha 360/158 dataset and SFM effect on Alpha 360 after fixing the bugs

### CODE REVIEW

File: examples/highfreq/high_freq_tree.ipynb [hunk 1]
```diff
@@ -0,0 +1,192 @@
+{
```
Comment (you-n-g) [inline examples/highfreq/high_freq_tree.ipynb:1]: Why is this file necessary in this PR?

File: examples/highfreq/highfreq_gdbt_model.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# Copyright (c) Microsoft Corporation.
```
Comment (you-n-g) [inline examples/highfreq/highfreq_gdbt_model.py:1]: Why is this file necessary in this PR?

File: examples/highfreq/workflow_config_High_Freq_Tree_Alpha158.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+qlib_init:
```
Comment (you-n-g) [inline examples/highfreq/workflow_config_High_Freq_Tree_Alpha158.yaml:1]: Why is this file necessary in this PR?

File: qlib/contrib/model/highfreq_gdbt_model.py [hunk 4]
```diff
@@ -0,0 +1,157 @@
+# Copyright (c) Microsoft Corporation.
```
Comment (you-n-g) [inline qlib/contrib/model/highfreq_gdbt_model.py:1]: Why is this file necessary in this PR?

File: qlib/workflow/record_temp.py [hunk 5]
```diff
@@ -155,7 +155,87 @@ def list(self):
     def load(self, name="pred.pkl"):
         return super().load(name)
 
+class HFSignalRecord(SignalRecord):
+    """
```
Comment (you-n-g) [inline qlib/workflow/record_temp.py:159]: Why is this file necessary in this PR?

File: examples/benchmarks/README.md [hunk 6]
```diff
@@ -17,6 +17,9 @@ The numbers shown below demonstrate the performance of the entire `workflow` of
 | ALSTM (Yao Qin, et al.) | Alpha360 | 0.0493±0.01 | 0.3778±0.06| 0.0585±0.00 | 0.4606±0.04 | 0.0513±0.03 | 0.6727±0.38| -0.1085±0.02 |
 | GATs (Petar Velickovic, et al.) | Alpha360 | 0.0475±0.00 | 0.3515±0.02| 0.0592±0.00 | 0.4585±0.01 | 0.0876±0.02 | 1.1513±0.27| -0.0795±0.02 |
 | DoubleEnsemble (Chuheng Zhang, et al.) | Alpha360 | 0.0407±0.00| 0.3053±0.00 | 0.0490±0.00 | 0.3840±0.00 | 0.0380±0.02 | 0.5000±0.21 | -0.0984±0.02 |
+| SFM (Liheng Zhang, et al.)| Alpha360 | 0.0314±0.00 | 0.2389±0.02| 0.0407±0.00 | 0.3245±0.01 | 0.0127±0.01 | 0.1440±0.07| -0.1631±0.01 |
```
Comment (you-n-g) [inline examples/benchmarks/README.md:20]: How many experiments have been run before you get this result ?
image

**Comment by you-n-g** [comment]
@javaThonc  Besides, please resolve the conflicts.

File: examples/benchmarks/README.md (hunk 6 above)
Comment (javaThonc) [reply examples/benchmarks/README.md:20]: I ran it 5 times. I will run the model again and update it.
